components/web/ejabberd/Makefile

```diff
@@ -65,41 +65,30 @@ COMPONENT_PRE_CONFIGURE_ACTION+= \
 COMPONENT_PRE_BUILD_ACTION+= \
   $(CLONEY) $(SOURCE_DIR) $(@D)
 
-CONFIGURE_OPTIONS +=	--sysconfdir=/etc
-CONFIGURE_OPTIONS +=	--with-erlang=$(USRBINDIR)
-CONFIGURE_OPTIONS +=	--localstatedir=/var
-CONFIGURE_OPTIONS +=	--enable-odbc
-CONFIGURE_OPTIONS +=	--enable-mysql
-CONFIGURE_OPTIONS +=	--enable-pgsql
-CONFIGURE_OPTIONS +=	--enable-sqlite
-CONFIGURE_OPTIONS +=	--enable-pam
-CONFIGURE_OPTIONS +=	--enable-riak
-CONFIGURE_OPTIONS +=	--enable-redis
-CONFIGURE_OPTIONS +=	--enable-new-sql-schema
-CONFIGURE_OPTIONS +=	--enable-sip
-CONFIGURE_OPTIONS +=	--enable-tools
+CONFIGURE_OPTIONS += --sysconfdir=$(ETCDIR)
+CONFIGURE_OPTIONS += --with-erlang=$(USRBINDIR)
+CONFIGURE_OPTIONS += --localstatedir=$(VARDIR)
+CONFIGURE_OPTIONS += --enable-odbc
+CONFIGURE_OPTIONS += --enable-mysql
+CONFIGURE_OPTIONS += --enable-pgsql
+CONFIGURE_OPTIONS += --enable-sqlite
+CONFIGURE_OPTIONS += --enable-pam
+CONFIGURE_OPTIONS += --enable-redis
+CONFIGURE_OPTIONS += --enable-new-sql-schema
+CONFIGURE_OPTIONS += --enable-sip
+CONFIGURE_OPTIONS += --enable-tools
 
 CONFIGURE_SBINDIR.64 = $(CONFIGURE_PREFIX)/bin
 
 COMPONENT_BUILD_TARGETS = all
 
-COMPONENT_TEST_ARGS = "CT_BACKENDS=mnesia"
-COMPONENT_TEST_TARGETS = test
-COMPONENT_TEST_TRANSFORMS= \
-    '-n ' \
-    '-e "/TEST COMPLETE/p"'
-
+REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
+REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
 REQUIRED_PACKAGES += SUNWcs
```

Comment:
Did you run gmake REQUIRED_PACKAGES or did you manually edit the Makefile?
I miss
# Auto-generated dependencies
in front of the required packages list which is automatically generated.